content/docs/sdk/flutter/at_client_manager.md

```diff
@@ -32,13 +32,23 @@ To access CRUD methods,
 AtClient atClient = atClientManager.atClient;
 ```
 
+**NotificationService**
+
+Notification service gives back stream of notifications from the server to the subscribing client.
+
 To access NotificationService methods 
 
 ```
 NotificationService notificationService = atClientManager.notificationService;
 ```
 
-And for the SyncService which is responsible for invoking the sync :
+**SyncService**
+
+SyncService syncs local secondary and cloud secondary.
```

Comment:
we can mention client app/mobile app instead of local secondary.
replace cloud secondary with remote secondary server